Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -386,42 +396,43 @@ public class ScrollEvent : UnityEvent<GameObject> { }
         /// Event that is fired on the target object when the ScrollingObjectCollection deems event as a Click.
         /// </summary>
         [Tooltip("Event that is fired on the target object when the ScrollingObjectCollection deems event as a Click.")]
-        public ScrollEvent ClickEvent = new ScrollEvent();
+        public ScrollEvent OnClick = new ScrollEvent();
 
         /// <summary>
         /// Event that is fired on the target object when the ScrollingObjectCollection is touched.
         /// </summary>
         [Tooltip("Event that is fired on the target object when the ScrollingObjectCollection is touched.")]
-        public ScrollEvent TouchStarted = new ScrollEvent();
+        public ScrollEvent OnTouchStarted = new ScrollEvent();
 
         /// <summary>
         /// Event that is fired on the target object when the ScrollingObjectCollection is no longer touched.
         /// </summary>
         [Tooltip("Event that is fired on the target object when the ScrollingObjectCollection is no longer touched.")]
-        public ScrollEvent TouchEnded = new ScrollEvent();
+        public ScrollEvent OnTouchEnded = new ScrollEvent();
 
         /// <summary>
         /// Event that is fired on the target object when the ScrollingObjectCollection is no longer in motion from velocity
         /// </summary>
         [Tooltip("Event that is fired on the target object when the ScrollingObjectCollection is no longer in motion from velocity.")]
-        public UnityEvent ListMomentumEnded = new UnityEvent();
+        public UnityEvent OnMomentumEnded = new UnityEvent();
 
         /// <summary>
         /// Event that is fired on the target object when the ScrollingObjectCollection is starting motion with velocity.
         /// </summary>
         [Tooltip("Event that is fired on the target object when the ScrollingObjectCollection is starting motion with velocity.")]
-        public UnityEvent ListMomentumBegin = new UnityEvent();
+        public UnityEvent OnMomentumStarted = new UnityEvent();
 
         [SerializeField]
         [HideInInspector]
         private CameraEventRouter cameraMethods;
 
         // Maximum amount the scroller can travel (vertically)
-        private float maxY
+        private float MaxY
         {
             get
             {
-                return NodeList.Count != 0 ? Mathf.Ceil(SafeDivisionFloat(NodeList.Count - (ViewableArea * ItemsPerTier), ItemsPerTier)) * CellHeight : 0.0f;
+                return contentMaxBounds == null || contentMaxBounds.size.y <= 0 ? 0 :
+                    Mathf.Max(0, SafeDivisionFloat(contentMaxBounds.size.y, transform.lossyScale.y) - TiersPerPage * CellHeight);
```

Comment:
doesn't this evaluate the null check in the front outside your condition and then cast the bool to float? :D 

for the sake of readability can we add some parenthesis

return (contentMaxBounds == null || contentMaxBounds.size.y <= 0) ? 0 :
                    Mathf.Max(0, SafeDivisionFloat(contentMaxBounds.size.y, transform.lossyScale.y) - TiersPerPage * CellHeight);